src/views/admin/PersonManagementView.vue

```diff
@@ -117,18 +114,9 @@
 
   function autoSelectSchule(): void {
     // Autoselect the Schule for the current user that only has 1 Schule assigned to him.
-    const personenkontexte: Array<UserinfoPersonenkontext> | null = authStore.currentUser?.personenkontexte || [];
-
-    if (personenkontexte.length > 0) {
-      const matchingOrganisations: UserinfoPersonenkontext[] = personenkontexte.filter(
-        (kontext: UserinfoPersonenkontext) =>
-          organisationStore.allOrganisationen.some((org: Organisation) => org.id === kontext.organisationsId),
-      );
-      if (matchingOrganisations.length === 1) {
-        const matchedOrganisation: UserinfoPersonenkontext | undefined = matchingOrganisations[0];
-        selectedSchulen.value = [matchedOrganisation?.organisationsId || ''];
-        hasAutoselectedSchule.value = true;
-      }
+    if (organisationStore.allOrganisationen.length === 1) {
+      selectedSchulen.value = [organisationStore.allOrganisationen[0]?.id || ''];
+      hasAutoselectedSchule.value = true;
```

Comment:
Ich denke, es ist nicht sicher, einfach dem aktuellen Benutzer die einzige Organisation zuzuweisen, die wir vom Endpunkt erhalten. Ist es nicht besser erstmal sicherstellen, dass der Current User eine einzige Organisation hat, die eine andere 

Ist es nicht besser, zunächst die Organisationen zu überprüfen, die der aktuelle Benutzer in currentUser.kontexte hat, bevor eine Übereinstimmung vom Endpunkt getAllOrganisations gefunden wird?

Reply:
Dadurch, dass wir bei der Abfrage nur die Organisationen filtern, auf die er das Systemrecht "..._VERWALTEN" hat, sollten nur die editierbaren Orgas kommen oder? Ist es dann nicht trotzdem sicher? Wollen wir das noch mal mit den Architekten besprechen? Ansonsten gebe ich dir recht, es muss sichergestellt sein und im Notfall vergleichen wir es dann mit dem Kontext

Reply:
Das Prüfen auf .._VERWALTEN bringt eine Schicht von Sicherheit ja... Reicht glaube Ich auch ohne Kontextprüfung in diesem Fall